dataset/src/test/scala/frameless/functions/AggregateFunctionsTests.scala

```diff
@@ -420,18 +415,15 @@ class AggregateFunctionsTests extends TypedDatasetSuite {
     evCanBeDoubleA: CatalystCast[A, Double]
   ): Prop = {
 
-
     val tds = TypedDataset.create(xs)
-    //typed implementation of bivar stats function
+    // Typed implementation of bivar stats function
```

Comment:
That should probably be "univariate stats function" instead

Reply:
indeed. fixed